.travis.yml

```diff
@@ -1,16 +1,19 @@
 language: node_js
 node_js:
-- '0.12'
+- '0.11'
 git:
   depth: 10
 before_install:
-- export DISPLAY=:99.0
-- sh -e /etc/init.d/xvfb start
 - npm install -qg bower gulp-cli
 - npm install -q
 - bower install --force
 - bower install --force
-script: gulp
+script: gulp ci
+sudo: false
```

Comment:
This is to stop the nag message about using Travis' new [container infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/)